docs/DafnyRef/Attributes.md

```diff
@@ -427,6 +426,14 @@ In the first and only verification round, this option will split the original [a
 into one assertion batch per assertion.
 This is mostly helpful for debugging which assertion is taking the most time to prove, e.g. to profile them.
 
+### 22.2.21. `{:options OPT0, OPT1, ... }` {#sec-attr-options}
```

Comment:
Section number `22.2.21` is used twice. (Here's an opportunity to claim section number `22.2.22`. :))